docs/docs/developers/guides/rpc_test_snapshot.md

```diff
@@ -0,0 +1,32 @@
+---
+title: RPC Test Snapshots
+sidebar_position: 2
+---
+
+This document describes how to generate and use RPC test snapshots. An RPC test snapshot contains the name of the RPC method, the request payload, the desired response, and the minimal database entries for replaying the request. Thus, it can be used as a regression test or a unit test for an RPC method.
+
+### Generate RPC test dumps
+
+An RPC test dump contains the name of the RPC method, the request payload, and the response of an RPC call. It is used to dump a minimal database snapshot and replay the request in a subsequent process.
+
+To generate test dumps, add `--dump-dir [PATH]` to `forest-tool api compare` command. e.g., `forest-tool api compare forest_snapshot_calibnet_2025-01-15_height_2320334.forest.car.zst --dump-dir /var/tmp/rpc-dump`
```

Comment:
Let's note that a running instance of Forest and Lotus are required, the latter requiring some environmental variables set (refer to docker compose file).

Reply:
Done.

Reply:
@LesnyRumcajs please re-approve if the change look good, thanks!